.github/workflows/build.yml

```
@@ -0,0 +1,27 @@
+        with:
+          node-version: 18
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: |
+            - recursive: true
+      - name: Lint
+        run: pnpm lint
+      - name: build
+        run: pnpm build
+      - name: publish
+        run: pnpm pkg-pr-new publish --package-manager=pnpm --pnpm
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)